src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const adder = (...number) => {
+    if (!number.length) {
+      const sumCopy = sum;
+
+      sum = 0;
+
+      return sumCopy;
+    } else {
+      sum += number[0];
+    }
```

Comment:
```suggestion
    } 
    
    sum += number[0];
```
You don't need else as you `return` in the `if`